public/locales/en/app.ftl

```diff
@@ -0,0 +1,35 @@
+-product-name = Firefox Monitor
+
+layout-support = Support
+layout-feedback = Give Feedback
+layout-terms = Terms&nbsp;and&nbsp;Privacy
+layout-copyright = Portions of this content are &copy; 1998-2018 by individual mozilla.org contributors. <br />Content available under a <a {{> analytics/link_event eventLabel="Creative Commons"}} href="https://www.mozilla.org/foundation/licensing/website-content/" target="_blank" rel="noopener">Creative Commons license</a>.
+
+error-not-subscribed = This email address is not subscribed to {-product-name}.
+error-hibp-throttled = Too many connections to HIBP.
+error-hibp-connect = Error connecting to HIBP.
```

Comment:
Not sure how visible these errors are, or if we should use a more verbose "Have I Been Pwned" vs the abbrev.

Reply:
In any case, HIBP needs an explanation in a comment.

Reply:
Side note: looks like there's a term for HIBP now, it should be used instead.